### PR TITLE
Feat/release and promote hw observer

### PR DESCRIPTION
**Pull request by jneo8**
Follow: #19 and #21

### CODE REVIEW

**Review by Pjack — APPROVED** [review]
It's good to me to apply to hardware observer.

**Comment by chanchiwai-ray** [comment]
https://github.com/canonical/solutions-engineering-automation/actions/runs/9315533909/job/25641883492?pr=22#step:6:89 this does not look good

**Comment by jneo8** [comment]
> https://github.com/canonical/solutions-engineering-automation/actions/runs/9315533909/job/25641883492?pr=22#step:6:89 this does not look good. (p.s. link is not working, I am referring "Terraform plan" step, line 85. The description of the repository is removed)



> https://github.com/canonical/solutions-engineering-automation/actions/runs/9315533909/job/25641883492?pr=22#step:6:89 this does not look good. (p.s. link is not working, I am referring "Terraform plan" step, line 85. The description of the repository is removed)

Nice catch, let me fix it.

**Review by chanchiwai-ray — APPROVED** [review]

**Review by chanchiwai-ray — APPROVED** [review]

**Review by Pjack — APPROVED** [review]

File: terraform-plans/files/github/charm_promote.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Promote charm to other tracks and channels
```
[the remark this answers is not in the document]
Comment (jneo8) [reply terraform-plans/files/github/charm_promote.yaml:1]: Yes, sure. Maybe add something like you should not change this by yourself warning?

[the remark this answers is not in the document]
Comment (jneo8) [reply terraform-plans/files/github/charm_promote.yaml:1]: Here is the [issue](https://github.com/canonical/solutions-engineering-automation/issues/28)

**Pull request merged**